### DISCUSSION TITLE
When could we expect a new extension release?

### DISCUSSION THREAD

**Discussion by RaczeQ** [duckdb/duckdb-spatial#116] [Q&A] [answered]
Hi,

I'm interested in using new features (#108) of `spatial` extension in the current `duckdb` version. The version currently available includes #90 PR.

Do you know when is the plan to release some new version on production?

**Comment by Maxxen** [comment, marked as answer]
Hi! The latest duckdb nightly builds should provide spatial built against commit #110 (dc66594). 

Since there has been breaking changes in core DuckDB between the last stable release (0.8.1) we're probably not going to issue a backport for it. There is a new major release (0.9.0) scheduled for early September though which will have an accompanying spatial release including all the fixes and features up to that point.

We're thinking of how to handle this going forward, but I think after 0.9.0 we will at least attempt to build for the latest nightly and the last stable release as long as possible until the diff in core duckdb inevitable becomes too big.

**Reply by NathanEmb** [reply]
Not to raise a dead thread but DuckDB 0.9.0 is released, has there been an accompanying spatial release? Currently spatial extension appears to be locked on DuckDB v0.7.1 according to releases tab and what happens when we tried to install and load the extension into native DuckDB 0.9.0

**Reply by Maxxen** [reply]
Hi! I think there should be binaries distributed for v0.9.0 (and new ones should be available soon for 0.9.1 as we build spatial nightly), I seem to be able to install and load spatial from the CLI (on arm64 macOS). What platform are you on? 

<img width="464" alt="image" src="https://github.com/duckdblabs/duckdb_spatial/assets/1337636/195d69cd-31d0-4c27-b2fa-838e2c5b4baa">

**Reply by NathanEmb** [reply]
Testing on Windows.

Originally downloaded from https://github.com/duckdblabs/duckdb_spatial/releases/tag/v0.0.2-alpha win x64 zip and then tried to build locally.

Then interacting through python-api 
```
import duckdb
conn = duckdb.connect('test.db', config={'allow_unsigned_extensions': True})
conn.sql("INSTALL 'vendor\\spatial.duckdb_extension'; LOAD spatial;")
```

resulted in 

```
InvalidInputException: Invalid Input Error: Extension "C:\Users\_____\.duckdb\extensions\v0.9.1\windows_amd64\spatial.duckdb_extension" version (v0.7.1) does not match DuckDB version (v0.9.1)
```

Tried to download the nightly for win x64 zip but it doesn't seem to be there. New to DuckDB and new to extensions so just trying to get my feet wet. Not using CLI/INSTALL b/c  I need local build only capability

Edit: made clarification

**Reply by Maxxen** [reply]
ahh ok I see, I've set it up so that we build for the latest duckdb and the latest stable version (as of today v0.9.1) in #137, as soon as it gets merged and CI finishes and I get into the office tomorrow Ill cut a new release here on GitHub so you can download the artifacts from here. Thanks for bringing this up, we've been so focused on distributing extension through the `INSTALL` command lately I forgot that that's not always an option.

**Reply by Maxxen** [reply]
New release is up! https://github.com/duckdblabs/duckdb_spatial/releases/tag/v0.9.1

**Reply by NathanEmb** [reply]
Thank you! Really appreciate the quick-turnaround.
